app/code/Magento/Cms/Test/Mftf/ActionGroup/AdminFillCmsBlockFormActionGroup.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="FillOutBlockContentWithMarginalSpace">
```

Comment:
Hi @ajithkumar-maragathavel. Thank you for your changes. There's one minor adjustment left. The action group name should equal the file name. You named the file correctly but forgot to rename the action group itself.

Reply:
@rogyar I have made the changes you suggested. Please review it.
Commit ID - 034f22e120db8332cc7d435a65c20e1279318df7